### ISSUE TITLE
broken desktop interface

### ISSUE THREAD

**Issue by dest4131** [diaspora/diaspora#7225]
I updated from 0.6.0 to 0.6.1.
The /stream interface is a bit broken. Messages are not in white background but in grey, as in page background.
I also see

>  #NSFW | Montrer le message | Tout montrer
> #NSFW | Masquer les messages #nsfw

at the top of every message.
Long messages overflow on the next messages, there is no "expand" function anymore.

In the console, I see several errors with CSP, such as 

> Content Security Policy: Ignoring “'unsafe-inline'” within script-src or style-src: nonce-source or hash-source specified

 for Firefox 50

and

> The Content Security Policy 'default-src 'none'; child-src 'self' www.youtube.com w.soundcloud.com twitter.com platform.twitter.com syndication.twitter.com player.vimeo.com www.mixcloud.com www.dailymotion.com media.ccc.de bandcamp.com www.instagram.com; connect-src 'self' embedr.flickr.com geo.query.yahoo.com nominatim.openstreetmap.org api.github.com; font-src 'self'; form-action 'self' platform.twitter.com syndication.twitter.com; frame-ancestors 'self'; img-src data: *; media-src https:; script-src 'self' 'unsafe-eval' platform.twitter.com cdn.syndication.twimg.com widgets.flickr.com embedr.flickr.com platform.instagram.com 'unsafe-inline' 'nonce-wB01bQF7s9WGVp+nEXN5bXIOfvHI3wmIttBqnFBbirU='; style-src 'self' 'unsafe-inline' platform.twitter.com *.twimg.com' was delivered in report-only mode, but does not specify a 'report-uri'; the policy will have no effect. Please either add a 'report-uri' directive, or deliver the policy via the 'Content-Security-Policy' header.

for Chromium 53.

I don't know if it is related. It appears on pages with the problem and on pages that are fine. Surprisingly, the interface works well when I display a specific message or if I navigate in the settings.

I don't see any (relevant) 404 in the console.

Does anybody have a clue about what is happening here? Sorry if it is a duplicate of a previous issue, I could not find any.

**Comment by SuperTux88** [comment]
The CSP warnings are OK, that doesn't break anything if it is in report-only mode (which it is by default).

About the broken interface:
* Can you maybe add a screenshot?
* Did you precompile the assets?

**Comment by dest4131** [comment]
Hi and thanks for your answer. Here are two screenshots, edited for privacy:
https://imgur.com/a/i6M4i

Yes I did precompile the assets with

> RAILS_ENV=production bin/rake assets:precompile

Doing it again only gives the following output:

> Rack::SSL is enabled


And I did a ctrl-F5 in the browser to refresh everything.
Notice my pod's code version is 0.6.1.0-pb88f53a3
Problem observed even with uBlock Origin, HTTPS everywhere and Noscript disabled.
